jobs/update_orphaning_dashboard_etl.py

```diff
@@ -1188,16 +1186,15 @@ def has_update_apply_failure_mapper(d):
 # Save the output to be uploaded automatically once the job completes.
 # The file will be stored at:
 # * https://analysis-output.telemetry.mozilla.org/app-update/data/out-of-date/FILENAME
```

Comment:
This will need to be updated.

Reply:
I retract that, reading the document attached in the bug it looks like this url is now backed by both s3 and gcs buckets.